assets/js/solcast.webflow.script.js

```diff
@@ -193,4 +185,46 @@ $(".slider-main_component").each(function (index) {
       disabledClass: "is-disabled"
     },
   });
-});
\ No newline at end of file
+});
+
+/*--------------------------*/
+/* Hight Light Text Script  */
+/*--------------------------*/
+
+document.addEventListener('DOMContentLoaded', function() {
+  // Select all elements with the class 'script-text-highlight'
+  var elements = document.querySelectorAll('.script-text-highlight');
```

Comment:
@Rework-Digital - can this highlight code go in a different script?

I.e. we shouldn't be dumping everything into a single giant assets/js/solcast.webflow.script.js if it only apples to certain pages/elements. 

This one only applies to the slider, so can it go into a separate script file that is only applied to the pages with the slider on it?

Reply:
I have split up the scipts